src/diffusers/loaders/controlnet.py

```diff
@@ -38,6 +38,9 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
                     - A link to the `.ckpt` file (for example
                       `"https://huggingface.co/<repo_id>/blob/main/<path_to_file>.ckpt"`) on the Hub.
                     - A path to a *file* containing all pipeline weights.
+            config_file (`str`, *optional*):
+                Filepath to the configuration YAML file associated with the model. If not provided it will default to:
+                https://raw.githubusercontent.com/lllyasviel/ControlNet/main/models/cldm_v15.yaml
```

Comment:
I don't think it makes the difference between `original_config_file` and `config_file` quite clear. Could you rephrase?

Reply:
They both mean the same thing. We used `original_config_file` in the pipeline Mixin. 

With the refactor, the internal logic `config_file = kwargs.pop("config_file")` was removed in favour of using `original_config_file` like we do for the Pipelines. 

But UI's like SD.Next use `config_file` when loading the VAE.  Putting it back here again for backwards compatibility. 

We don't need to keep both. We can deprecate one of them.

Reply:
Okay. Then maybe in a follow-up PR, let's start their deprecation cycles?